Framework/CorDebug/CorDebugClass.cs

```diff
@@ -122,7 +122,7 @@ public uint TypeDef_Index
         int ICorDebugClass2.GetParameterizedType( CorElementType elementType, uint nTypeArgs, ICorDebugType []ppTypeArgs, out ICorDebugType ppType )
         {
             // CorDebugClass.GetParameterizedType is not implemented

```

Comment:
Technically, I guess this comment isn't true anymore.  It is implemented, but only works for non-generic types (which is the only kind of type that ever gets requested)